app/js/actions/actionTypes.js

```diff
@@ -47,3 +47,6 @@ export const DELETE_ALL_DRAFT_LAB_ORDERS = 'DELETE_ALL_DRAFT_LAB_ORDERS';
 export const SET_ORDER_ACTION = 'SET_ORDER_ACTION';
 export const GET_DATE_SUCCESS = 'GET_DATE_SUCCESS';
 export const GET_DATE_FAILURE = 'GET_DATE_FAILURE';
+export const CREATE_LAB_ORDER = 'CREATE_LAB_ORDER';
+export const CREATE_LAB_ORDER_SUCCESS = 'CREATE_LAB_ORDER_SUCCESS';
```

Comment:
oh, good point.

but my thought process was that since the resource is being created, it's better to use `CREATE_LAB_ORDER` as it was more specific while `SAVE_LAB_ORDER` could mean updating or creating. but I think `SAVE_DRAFT_LAB_ORDER` makes more sense though. thanks.

I'll rectify this.